re_pages/Packages.res

```diff
@@ -490,6 +500,7 @@ let default = (props: props) => {
   )
 
   let onKeywordSelect = keyword => {
+    updateQuery(keyword)
     scrollToTop()
```

Comment:
following up on my previous comment regarding side-effects, I guess I should have put the scrollToTop into the useEffect as well